e2e-tests/spring-rest-openapi-v2/src/test/java/com/foo/rest/examples/spring/adaptivehypermutation/AHypermutationRestController.java

```diff
@@ -0,0 +1,141 @@
+package com.foo.rest.examples.spring.adaptivehypermutation;
+
+import com.p6spy.engine.spy.P6SpyDriver;
+import kotlin.random.Random;
+import org.evomaster.client.java.controller.EmbeddedSutController;
+import org.evomaster.client.java.controller.InstrumentedSutStarter;
+import org.evomaster.client.java.controller.api.dto.AuthenticationDto;
+import org.evomaster.client.java.controller.api.dto.SutInfoDto;
+import org.evomaster.client.java.controller.db.DbCleaner;
+import org.evomaster.client.java.controller.problem.ProblemInfo;
+import org.evomaster.client.java.controller.problem.RestProblem;
+import org.hibernate.dialect.H2Dialect;
+import org.springframework.boot.SpringApplication;
+import org.springframework.context.ConfigurableApplicationContext;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.sql.Connection;
+import java.sql.SQLException;
+import java.util.List;
+import java.util.Map;
+
+/** automatically created on 2020-10-22 */
+public class AHypermutationRestController extends EmbeddedSutController {
```

Comment:
why is this not extending `SpringWithDbController` ?

Reply:
thanks
I directly used the controller from a separate project. This is fixed now.